sample_code/ep138/rev01/t.py

```diff
@@ -0,0 +1,28 @@
+
+
+f = open('filename', 'w')
+try:
+    f.write('hello hello world\n')
+finally:
+    f.close()
+
+
+with (
+        open('filename', 'w') as f,
+        open('filename2', 'w') as f2,
+):
```

Comment:
This specific example might be confusing for ppl, because it produces `SyntaxError` for older python versions (hence the flake8 ignore).
But at the same time -- it is a really good example and shows best practice with multiple manages (despite the 3.9+ syntax limitation).

Idk, let me know if you want me to remove it or leave it there :)

Reply:
probably in 2022 when pre-commit.ci upgrades the image I'll sweep through and fix this up